README.md

```diff
@@ -1 +1,81 @@
-# th2-conn-ws-client
\ No newline at end of file
+# WebSocket Client v0.0.1
+
+This microservice allows sending and receiving messages via WebSocket protocol
+
+## Configuration
+
+The main configuration is done by changing following properties:
+
++ **uri** - connection URI
++ **sessionAlias** - session alias for incoming/outgoing th2 messages (e.g. `ws_api`)
++ **handlerSettings** - WebSocket event handler settings
+
+### Event handler configuration
+
+Event handler can be configured by changing following properties in the `handlerSettings` block of the main configuration:
+
++ **pingInterval** - interval for sending ping-messages in ms (`30000` by default)
+
+### Configuration example
+
+```yaml
+uri: wss://echo.websocket.org
+sessionAlias: api_session
+handlerSettings:
+  pingInterval: 30000
+```
+
+### MQ pins
+
+* input queue with `subscribe` and `send` attributes for outgoing messages
+* output queue with `publish`, `first` (for incoming messages) or `second` (for outgoing messages) attributes
+
+## Inputs/outputs
+
+This section describes the messages received and produced by the service
+
+### Inputs
+
+This service receives messages to be sent via MQ as `MessageGroup`s containing a single `RawMessage` with a message body
+
+### Outputs
+
+Incoming and outgoing messages are sent via MQ as `MessageGroup`s containing a single `RawMessage` with a message body.
```

Comment:
We can also add a comma before the word 'containing' as mentioned in the previous comment.